lab-maddy/route/route-toy.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const Toy = require('../model/toy');
+const storage = require('../lib/storage');
+const response = require('../lib/response');
+const debug = require('debug')('http:route-toy');
+
+module.exports = function(router) {
+  router.post('/api/toy', (req, res) => {
+    debug('/api/toy POST');
+    try {
+      let newToy = new Toy(req.body.name, req.body.desc, req.body.price, req.body.material);
```

Comment:
I think you would want to move this line into the storage module and then do some validation in there before creating a new object. As it is right now, it will create an object with just an _id property.